pyspark/bigdl/examples/keras/mnist_cnn.py

```diff
@@ -48,38 +56,27 @@ def build_keras_model():
     keras_model.add(Activation('relu'))
     keras_model.add(MaxPooling2D(pool_size=(2, 2)))
     keras_model.add(Dropout(0.25))
-
     keras_model.add(Flatten())
     keras_model.add(Dense(128))
     keras_model.add(Activation('relu'))
     keras_model.add(Dropout(0.5))
     keras_model.add(Dense(10))
     keras_model.add(Activation('softmax'))
-
     return keras_model
 
 
-def save_keras_model(keras_model, path):
-    """
-    Save a Keras model to JSON with given path
-    """
-    model_json = keras_model.to_json()
-    with open(path, "w") as json_file:
-        json_file.write(model_json)
-
-
 if __name__ == "__main__":
     keras_model = build_keras_model()
     def_path = "/tmp/lenet.json"
-    save_keras_model(keras_model, def_path)
+    save_keras_definition(keras_model, def_path)
```

Comment:
we should give an example that saving definition and also parameters.

Reply:
imdb example save definition and parameters